DependencyInjection/Compiler/StoragePass.php

```diff
@@ -139,8 +164,8 @@ private function overloadSuluMediaFormatCache(ContainerBuilder $container, array
     {
         $def = $container->getDefinition(self::SULU_MEDIA_FORMAT_CACHE_SERVICE_ID);
         $def->setClass(PBFormatCache::class)->setArguments([
-            new Reference(self::FILESYSTEM_PROVIDER_SERVICE_ID),
-            new Reference(self::FILE_RESOLVER_SERVICE_ID),
+            new Reference(self::FORMAT_CACHE_FILESYSTEM_PROVIDER_SERVICE_ID),
+            new Reference(self::FORMAT_CACHE_FILE_RESOLVER_SERVICE_ID),
```

Comment:
These two lines were actually incorrect — we need to inject a different filesystem provider and file resolver (which wraps the respective provider) into the format cache.